lib/contentful/asset.rb

```diff
@@ -44,19 +44,12 @@ def initialize(*)
     # @option options [String] :focus
     # @option options [String] :fit
     # @option options [String] :fl File Layering - 'progressive'
+    # @option options [String] :background
     # @see _ https://www.contentful.com/developers/documentation/content-delivery-api/#image-asset-resizing
     #
     # @return [String] Image API URL
     def image_url(options = {})
-      query = {
-        w: options[:w] || options[:width],
-        h: options[:h] || options[:height],
-        fm: options[:fm] || options[:format],
-        q: options[:q] || options[:quality],
-        f: options[:f] || options[:focus],
-        fit: options[:fit],
-        fl: options[:fl]
-      }.reject { |_k, v| v.nil? }
+      query = build_query(options)
```

Comment:
Had to move this into a separate method to fix rubocop error `Perceived complexity for image_url is too high.`